sdk/storage/Azure.Storage.Blobs/Changelog.txt

```diff
@@ -0,0 +1,11 @@
+Release History
```

Comment:
Should this and `BreakingChanges` be .md? does it matter?

Reply:
The Storage team has been using these as .txt forever.  I don't think it matters much and we can switch to .md in the future, but we can't change it right now because it's referenced in their NuGet package description.